Cargo.toml

```diff
@@ -41,6 +41,8 @@ markdown-it-ruby = "1"
 markdown-it-block-spoiler = "1"
 markdown-it-footnote = "0.2"
 
+uuid = "=1.12.1"
```

Comment:
This is to address [the issue I brought up in the backend PR](https://github.com/LemmyNet/lemmy/issues/5457).

Reply:
Nice, ya much better to not have to change types in the back-end.